interface/resources/qml/controls/+webengine/FlickableWebViewCore.qml

```diff
@@ -136,8 +140,10 @@ Item {
             webChannel.registerObject("eventBridge", eventBridge);
             webChannel.registerObject("eventBridgeWrapper", eventBridgeWrapper);
 
-            if (webViewCoreUserAgent !== undefined) {
-                webViewCore.profile.httpUserAgent = webViewCoreUserAgent
+            if (flick.userAgent !== undefined) {
+                webViewCore.profile.httpUserAgent = flick.userAgent;
+                webViewCore.profile.offTheRecord = false;
+                webViewCore.profile.storageName = "qmlWebEngine";
```

Comment:
what do these two lines do?

Reply:
Since I am no longer using the HFWebEngineProfile class I had to manually do what they did (which wasn't much, except set an interceptor which we don't need at the moment.)

For example, off the record being true would make all things relevant to that engine be set in memory instead of stored. Probably for privacy reasons.

The storage location is mildly confusing due to its description in Qt but basically before setting these two I would have history for the web engine (maybe all web engines?) persist restarts. After setting these two, that no longer happens. Which I believe is intended functionality.

Eventually we can look into making it so web engines can have persistent storage whilst maintaining privacy.